tools/buildPackage.js

```diff
@@ -19,11 +19,24 @@ var env = {
   NODE_ENV: 'production'
 }
 
+console.log('Cleaning build target')
+
+if (isWindows) {
+  var cmds = [
+    'if exist ' + buildDir + ' rmdir /s /q ' + buildDir,
+  ]
+  execute(cmds, env, console.log.bind(null, 'done'))
+} else {
+  var cmds = [
+    'rm -rf ' + buildDir,
+    'rm -f dist/*.dmg dist/*.nupkg dist/*.exe dist/*.msi dist/RELEASES dist/*.zip',
+  ]
+  execute(cmds, env, console.log.bind(null, 'done'))
+}
```

Comment:
Ah, thanks! I just saw this after coming up with something similar (but
different). I can re-shape it to use append/push as appropriate. Let me
know!

On Sun, Jan 24, 2016 at 10:30 AM, Brian R. Bondy <notifications@github.com>
wrote:

> In tools/buildPackage.js
> <https://github.com/brave/browser-laptop/pull/299#discussion_r50639652>:
>
> > @@ -19,11 +19,24 @@ var env = {
> >    NODE_ENV: 'production'
> >  }
> >
> > +console.log('Cleaning build target')
> > +
> > +if (isWindows) {
> > +  var cmds = [
> > +    'if exist ' + buildDir + ' rmdir /s /q ' + buildDir,
> > +  ]
> > +  execute(cmds, env, console.log.bind(null, 'done'))
> > +} else {
> > +  var cmds = [
> > +    'rm -rf ' + buildDir,
> > +    'rm -f dist/*.dmg dist/*.nupkg dist/*.exe dist/*.msi dist/RELEASES dist/*.zip',
> > +  ]
> > +  execute(cmds, env, console.log.bind(null, 'done'))
> > +}
>
> Please do this:
>
> Put the var cmds = []
>
> At the top
>
> Then do cmds.push(...) for the remove call per platform.
> Then below where you removed the commands do:
> cmds = cmds.append([...])
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/brave/browser-laptop/pull/299/files#r50639652>.
>